docs/log-in-with-techpass.md

```diff
@@ -44,14 +44,14 @@ TechPass users who have a non-SE GSIB device can log in to TechPass portal.
 
 **To log in to TechPass portal**
 
-1. From your non-SE GSIB device, go to the [TechPass portal](http://portal.techpass.gov.sg/).
+1. Using your non-SE GSIB device, go to the [TechPass portal](http://portal.techpass.gov.sg/).
 2. Click **Login with TechPass**.
 3. Sign in to your TechPass account or choose the required TechPass account.
```

Comment:
do you mean "Sign in to your TechPass account by choosing the relevant TechPass account."?

Reply:
hmm, this situation arises when user has more than one techpass account, so i had mentioned "choose the required."